canonical-web-and-design/www.ubuntu.com
# Add fix for marketo renaming of g-captcha-response

## Done

- Added js that creates a new field called grecaptcharesponse to hold the inforomation in g-recaptcha-response, because Marketo secretly doesn't allow hyphens

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack#get-in-touch) or any other marketo form
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Fill in the form and check the recaptcha box and see that it gets to marketo and that the marketo webhook tests the submission


## Issue / Card

Fixes #4888

## Screenshot

![image](https://user-images.githubusercontent.com/441217/55193029-cd719880-519d-11e9-9345-4d0fa999b25a.png)
